### DISCUSSION TITLE
How to backup

### DISCUSSION THREAD

**Discussion by sgrodnik** [wg-easy/wg-easy#230] [Q&A] [answered]
Hi, tell me please how to make a backup of my ~15 client configs in case of moving to the new server. I don't want to recreate new configs from scratch and resend them to users. I guess a user can just change IP in their current config. Is there any folder with configs? I am new to linux and docker.

**Comment by eandersons** [comment, marked as answer]
If you have defined volume for `/etc/wireguard` the same as in [`docker-compose.yml`](/WeeJeWel/wg-easy/blob/master/docker-compose.yml) then all configuration files (`wg0.conf` and `wg0.json`) which contain configuration for interface and peers should be located in the same directory as `docker-compose.yml`.

If a named volume is used then `/etc/wireguard/wg0.conf` and `/etc/wireguard/wg0.json` from container should be copied to a local file system (for example, by defining a temporary bind-mounted volume).
More information on volumes can be found on [Docker Compose file reference](https://docs.docker.com/compose/compose-file/compose-file-v3/#volumes).

**Comment by sgrodnik** [comment]
Thanks
Yes, you are right. Manual change `/etc/wireguard/wg0.json` and restarting (`docker stop wg-easy && docker rm wg-easy && docker pull weejewel/wg-easy && docker run -d \...`) do needed effect - web GUI refreshes accordingly. Great!

**Comment by rhbkweb** [comment]
Hi,

I want to make daily rsync backups of my wg-easy server, to be able to restore in case any problems happen.

About this two questions:

1- The only folder needed to backup is `/root/.wg-easy` that is where both `wg0.conf` and `wg0.json` files are located, correct?

2- To restore the backup to other server (with different IP address) follow the steps:

a) Make on the new server a new and clean wg-easy installation
b) Replace the `/root/.wg-easy` folder with the backup `(wg0.conf`, `wg0.json`)
c) Reboot server

Thanks
